mgmtsystem_extended_hr/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Marcelo Frare (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# Copyright 2019 Stefano Consolaro (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name":     "Management System - Extend HR",
+    "summary":  "Bridge module between mgmsystem and hr",
+    "version":  "11.0.1.0.0",
+
+    "author":   "Associazione PNLUG - Gruppo Odoo, Odoo Community Association (OCA)",
+    "website":  "https://gitlab.com/PNLUG/Odoo/management-system-improvements/tree/"
+                "11.0/mgmtsystem_extended",
+    "license":  "AGPL-3",
+
+    "category": "Management System",
+    "depends": ['hr',
+                'mgmtsystem_nonconformity'
+                ],
+    "data":    ['views/mgmtsystem_nonconformity_views.xml'
+                ],
```

Comment:
Just a suggestion: this is usually more readable and results in fewer changed lines in case of future commits:

```
'depends': [
    'hr',
    'mgmtsystem_nonconformity',
],
'data': [
    ...
],
```

Reply:
Thanks for these suggestion.